docs/devGuide/projectManagement.md

```diff
@@ -163,9 +129,16 @@ For general best practices, refer to the guide [_Working with PRs_ @SE-EDU](http
         <link rel="stylesheet" href="../markbind/css/bootstrap.min.css">
    ```
 
-   However, if there are any extra lines changed, that means that someone screwed up the functional tests, and needs to be fixed accordingly!
+   If any other lines changed, that likely means the functional tests weren't updated in a pull request. Do revert everything done so far, and make a pull request specifically to fix that first!
 
-1. **Combine** the changes that you made in the last two steps with the version commit generated by `npm version`. To do so, amend the version commit and version tag by doing the following (change `vA.B.C` to the new version's string accordingly).
+1. **If we did a new release for the Vue components**: also ensure that the minified `components.min.js` bundle from [earlier](#building-the-ui-components-library) is copied to each of the expected test site's folders.
+
+   <box type="info" seamless>
+
+   We do this for each MarkBind release (rather than update `components.min.js` for each affected PR), in order to reduce unnecessary merge conflicts. It also makes it easier for the maintainers to vet the changes.
```

Comment:
> Can this be a test case to ensure that components.min.js is not manually modified?

not too sure what you mean here, don't the functional tests already check for this? 😮

> Otherwise, the site for PRs and the master branch may not work as one may expect.
> This would also remove the manual work that maintainers need to do when releasing.

If I'm understanding this correctly, I think we should stick to a release cycle for the bundle. It's a little inconvenient during release, but helps give a more accurate amount of line diffs.
For netlify and such, perhaps we could add in a `npm run build:components` prior to build?

Reply:
> don't the functional tests already check for this?

Not if you update the test sites too. I mean to test that `components.min.js` was generated.

> but helps give a more accurate amount of line diffs. (during PR reviews)

What is this number used for?

> For netlify and such, perhaps we could add in a `npm run build:components` prior to build?

That sounds good.
But it's still inconvenient to test merged-but-unreleased features on local machine right?

Reply:
> What is this number used for?

I meant this, don't think having `+10, -20000` on every pr is a good idea:
![Untitled](https://user-images.githubusercontent.com/3306138/85199941-0909fc80-b326-11ea-80a2-3a387ab12f43.png)

> But it's still inconvenient to test merged-but-unreleased features on local machine right?

`npm run watch:components` should fix you up

Reply:
> Not if you update the test sites too. I mean to test that components.min.js was generated.

ah ok I see where you're coming from. hmm...

perhaps inside travis?
```
...
- cp pr-ed components.min.js to right beside it
- npm run build:components
- diff the two files
...
```

Reply:
> npm run watch:components should fix you up

once hot/live reload is setup, I aim to further streamline this into `dev` script of sorts, or maybe `markbind serve -d`. Can't say how it'll turn out exactly for now